### DISCUSSION TITLE
GitHub pages deployment action not working

### DISCUSSION THREAD

**Discussion by nestorSag** [importantimport/urara#75] [Q&A] [answered]
Hi, the GitHub actions workflow to deploy on GitHub pages is not working in a freshly forked repo. It does not crash, but it doesn't commit any files to the `gh-pages` branch. Looking at the logs, it does not find the `build` folder, even though the `build` step finishes succesfully, see logs from workflow below

```
  [INFO] ForceOrphan: false
  /usr/bin/git clone --depth=1 --single-branch --branch gh-pages ***github.com/nestorSag/urara-test.git /home/runner/actions_github_pages_1705849564243
  Cloning into '/home/runner/actions_github_pages_1705849564243'...
  [INFO] clean up /home/runner/actions_github_pages_1705849564243
  [INFO] chdir /home/runner/actions_github_pages_1705849564243
  /usr/bin/git rm -r --ignore-unmatch *
  rm '.nojekyll'
  [INFO] chdir /home/runner/actions_github_pages_1705849564243
  [INFO] prepare publishing assets
  [INFO] copy /home/runner/work/urara-test/urara-test/build to /home/runner/actions_github_pages_1705849564243
  cp: no such file or directory: /home/runner/work/urara-test/urara-test/build/*
  cp: no such file or directory: /home/runner/work/urara-test/urara-test/build/.*
  [INFO] delete excluded assets
  rm: no paths given
  [INFO] Created /home/runner/actions_github_pages_1705849564243/.nojekyll
```

the folder exists when I build locally, so I'm not sure what's wrong. Has anybody faced this?

**Comment by kwaa** [comment]
Hmm... This workflow hasn't been updated in a while, I might need to check it.

You can try to modify it yourself if needed: https://github.com/importantimport/.github/blob/main/.github/workflows/pnpm-gh-pages.yml

**Comment by nestorSag** [comment, marked as answer]
The problem was that no adapter was configured within the GH action. I solved this by setting the environment variable `ADAPTER` which is read in `svelte.config.js`
